acceleration-everest-03-dotnet/Api/Program.cs

```diff
@@ -20,12 +21,15 @@
         .UseMySql(connectionString, ServerVersion.AutoDetect(connectionString)));
 builder.Services.AddControllers();
 builder.Services.AddSwaggerGen();
+builder.Services.AddControllers();
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddFluentValidationAutoValidation();
 builder.Services.AddTransient<ICustomerService, CustomerService>();
 builder.Services.AddTransient<ICustomerAppService, CustomerAppService>();
-builder.Services.AddValidatorsFromAssembly(Assembly.Load(nameof(AppServices)));
 builder.Services.AddAutoMapper(Assembly.Load(nameof(AppServices)));
+builder.Services.AddScoped<DbContext, WarrenEverestDotnetDbContext>();
+builder.Services.AddUnitOfWork();
+builder.Services.AddUnitOfWork<WarrenEverestDotnetDbContext>();
```

Comment:
Pode utilizar somente o UnitOfWork referenciando o context.
```suggestion
builder.Services.AddUnitOfWork<WarrenEverestDotnetDbContext>();
```